docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,177 @@
+we plan to implement is [provided here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/pod-preemption.md#preemption-mechanics).
+
+#### PodDisruptionBudget is not supported
+[Pod Disruption Budget (PDB)](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/)
+allows application owners to limit the number pods of a replicated application that
+are down simultaneously from voluntary disruptions. However, alpha version of
+preemption does not respect PDB when choosing preemption victims.
+We plan to add PDB support in beta, but even in beta respecting PDB will be best
+effort. Scheduler will try to find victims whose
+PDB won't be violated by preemption, but if no such victims are found, preemption
+will still happen and lower priority pods will be removed despite their PDBs 
+being violated.
+
+#### Inter-Pod Affinity on Lower Priority Pods
+The current implementation of preemption considers a node for preemption only when
+the answer to this question is positive: "If all the pods with lower priority than
+the pending pod are removed from the node, can the pending pod be scheduled on
+the node?"
+
+If the answer is no, that node will not be considered for preemption. If the pending
+pod has inter-pod affinity on one or more of those lower priority pods on the node, the
+inter-pod affinity rule cannot be satisfied in the absence of the lower priority
+pods and scheduler will find the pending pod infeasible on the node. As a result,
+it will not try to preempt any pods on that node.
+Scheduler will try to find other nodes for preemption and could possibly find another
+one, but there is no guarantee that such a node will be found.
+
+We may address this issue in future versions, but we don't have a clear plan. Part
+of the reason is that finding the set of lower priority pods that satisfy all
+inter-pod affinity/anti-affinity rules is computationally expensive and adds
+substantial complexity to the preemption logic. Besides, even if preemption keeps the lower
+priority pods to satisfy inte-pod affinity, the lower priority pods may be preempted
+later by other pods, which removes the benefits of having the complex logic of 
+respecting inter-pod affinity to lower priority pods.
+
+Our recommended solution for this problem is to create inter-pod affinity towards
+equal or higher priority pods.
+
+#### Cross Node Preemption
+When considering a node N for preemption in order to schedule pending pod P,
+P may become feasible on N only when pods on other nodes are preempted. For 
+example, if there is anti-affinity from existing lower priority pods in a zone
+towards pod P, P may be scheduled in the zone only when those lower priority pods
+are preempted. Current preemption algorithm does not perform preemption of pods
+on nodes other than N, when considering N for preemption.
+
+We may consider adding cross node preemption in future versions if we find an
```

Comment:
Let's be explicit: "Fixing this will NOT be Beta or GA blocker".

Reply:
Again, I am a bit hesitant to expose users to our release planning in this kind of docs. I added a sentence to say that we cannot promise anything.